ttnotes.cls

```diff
@@ -56,3 +56,5 @@
 
 \newcommand{\ttrue}{\textsf{true}}
 \newcommand{\ffalse}{\textsf{false}}
+
+\newcommand{\subst}[2]{[\![#1 \setminus #2]\!]}
```

Comment:
mi atterrei alla notazione standard per la sostituzione, e quindi e[y/x] per sostituire y al posto di x in e, o ancora meglio e(y) se prima il termine era e(x). nel nostro caso la sostituzione diventa e(n, R(n, d, (x,y)e(x,y))).
eviterei le semantic brackets [[ ]] perche' usate in model theory/semantics con altri significati.